Sources/Navigator/EPUB/Assets/Static/scripts/readium-fixed.js

```diff
@@ -4212,6 +4212,91 @@ function initializeAccessibility(doubleTapLabel, energyBarLabel) {
       element.setAttribute("role", "button")
       element.setAttribute("aria-label", text)
     }
+    
+    /*
+     Given a locator, it will look for decorations (annotations + shared annotations) associated with that locator, so
+     it will animate the found HTML decorator, so, it will move to the right, then it will return to original position
+     */
+    function animateDecorationsFromLocator(locator) {
+        let sharedAnnotationsElements = getSharedAnnotationsItemsForLocator(locator)
+        let annotationsElements = getAnnotationsItemsForLocator(locator)
+        let elements = sharedAnnotationsElements.concat(annotationsElements)
+        elements.forEach(element => {
+            animate(element, 0, true);
```

Comment:
Stylistically we haven't been using the semicolon anywhere

Reply:
Fixed